### PR TITLE
Doc/#1069 restructure page doc

### PR DESCRIPTION
**Pull request by FlorianJacta**
Partially solves #1069 

# Pages

image

- How to create and define a page for a single-page application.
- And how to create a multi-page application

The length of the index page is reduced to show what a user needs to create, build, and define a page/pages only.

## Navigate between pages

image

How to navigate between pages: 4 ways to do so and each one with code examples.

## Partials

image

Tutorial on partials to explain what a partial is and how to use it.

## Dialog

image

I should provide code examples that work almost directly.

## Advanced

image

Non primary information on pages

### CODE REVIEW

File: docs/manuals/userman/auth/authorization.md [hunk 1]
```diff
@@ -201,7 +201,7 @@ Taipy GUI Enterprise edition uses role traits to select pages that user can disp
 The `AuthorizedPage^` class lets you specify a role traits filter where both the *success* and
 *failure* cases must be a page renderer (an instance of `Markdown^`,  `Html^`, or
 `(taipy.)gui.builder.Page^` instance - see the
-[section on Page Renderers](../gui/pages/index.md#defining-the-page-content) for more
+[section on Page Renderers](../gui/pages/index.md#building-page-content) for more
```
Comment (FabienLelaquais) [inline docs/manuals/userman/auth/authorization.md]:
```suggestion
[section on Page Renderers](../gui/pages/index.md#defining-the-page-content) for more
```

File: docs/manuals/userman/gui/pages/index.md [hunk 2]
```diff
@@ -49,313 +34,225 @@ Pages can be defined using two different techniques:
...
-associates a page with its name:
-```python
-...
-pages = {
-  "page1": Markdown("# My first page"),
-  "page2": Markdown("# My second page")
-}
-Gui(pages=pages)
-```
+## Building page content
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/index.md]:
```suggestion
## Defining the page content
```

File: docs/manuals/userman/gui/pages/index.md [hunk 3]
```diff
@@ -49,313 +34,225 @@ Pages can be defined using two different techniques:
...
 
-In this situation, to see the pages in your browser, the address you will use
-will be *localhost:5000/page1* or *localhost:5000/page2*.
+Building a page can consist of:
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/index.md]:
```suggestion
The definition of a page typically consists of:
```

File: docs/manuals/userman/gui/pages/index.md [hunk 4]
```diff
@@ -49,313 +34,225 @@ Pages can be defined using two different techniques:
...
 
-Note that if pages are created in different modules, the variables that they can bind
-to visual elements may have a scope limited to their origin module. See
-[Page scopes](../binding.md#scope-for-variable-binding) for more details.
+- Adding visual elements to the page.
+- Binding variables to these elements.
+- Setting properties to these elements.
+- Callbacks can be added to these elements to make your application interactive.
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/index.md]:
```suggestion
- Setting callbacks to these elements.
```

File: docs/manuals/userman/gui/pages/index.md [hunk 5]
```diff
@@ -49,313 +34,225 @@ Pages can be defined using two different techniques:
...
-       }
-       Gui(pages=pages).run()
-    ```
-    When you run this application and display the page at `http://127.0.0.1:5000/`,
-    you will notice that the browser navigates to the page `/page1`, and that the
-    final result is a page that contains the content of the root page, followed by
-    what is defined in the page `"page1"`.<br/>
-    In this example, you will see both the main title ('Multi-page application') and
-    the sub-title ('This is page 1').
+        <|{value}|slider|on_change=on_slider|>
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/index.md]:
```suggestion
        <|{value}|slider|on_change=slider_moved|>
```

File: docs/manuals/userman/gui/pages/index.md [hunk 6]
```diff
@@ -49,313 +34,225 @@ Pages can be defined using two different techniques:
...
 
-    If you navigate to '/page2', the main title remains on the page, and the sub-title
-    is replaced by the text 'This is page 2'
+        <|{data}|chart|>
+        """
 
-!!! tip "Running multiple services"
+        def on_slider(state):
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/index.md]:
```suggestion
        def slider_moved(state):
```

File: docs/manuals/userman/gui/pages/index.md [hunk 7]
```diff
@@ -49,313 +34,225 @@ Pages can be defined using two different techniques:
...
-       }
-       Gui(pages=pages).run()
-    ```
-    This application does the same as in the previous example, except that you now
-    have the footer line (*'This application was created...'*) in all the pages of
-    your application.
+        with tgb.Page() as page:
+            tgb.text(value="# Taipy Getting Started", mode="md")
+            tgb.text(value="Value: {value}")
+            tgb.slider(value="{value}", on_change=on_slider)
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/index.md:113]:
```suggestion
            tgb.slider(value="{value}", on_change=slider_moved)
```

File: docs/manuals/userman/gui/pages/index.md [hunk 8]
```diff
@@ -49,313 +34,225 @@ Pages can be defined using two different techniques:
...
 
-       <|content|>
+        def compute_data(decay:int)->list:
+            return [cos(i/6) * exp(-i*decay/600) for i in range(100)]
 
-       This application was created with [Taipy](https://www.taipy.io/).
-       """
-       page1_md="## This is page 1"
-       page2_md="## This is page 2"
+        def on_slider(state):
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/index.md:107]:
```suggestion
        def slider_moved(state):
```

File: docs/manuals/userman/gui/pages/index.md [hunk 9]
```diff
@@ -49,313 +34,225 @@ Pages can be defined using two different techniques:
...
 
-# Partials
+    with tgb.Page() as page:
+        tgb.text("# First Page", mode="md")
 
-There are page fragments that you may want to repeat on different pages. In that situation,
-you will want to use the *Partial* concept: a *Partial* is similar to a page (and built
-in a very similar way) that can be used multiple times in different visual elements.
-This prevents you from having to repeat yourself when creating your user interfaces.
+    Gui()
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/index.md]:
```suggestion
     Gui(page)
```

File: docs/manuals/userman/gui/pages/index.md [hunk 10]
```diff
@@ -49,313 +34,225 @@ Pages can be defined using two different techniques:
...
+    gui.add_page(page)
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/index.md]: Remove this line and the next

File: docs/manuals/userman/gui/pages/index.md [hunk 11]
```diff
@@ -49,313 +34,225 @@ Pages can be defined using two different techniques:
...
-       """
+    from taipy import Gui
 
-       Gui(page).run()
-    ```
+    page = "# First page"
 
-Please refer to the documentation page on the [`dialog`](../viselements/generic/dialog.md)
-control for more details and examples.
+    Gui(page=page).run()
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/index.md]:
```suggestion
    Gui(page).run()
```

File: docs/manuals/userman/gui/pages/index.md [hunk 12]
```diff
@@ -49,313 +34,225 @@ Pages can be defined using two different techniques:
...
+    gui.add_page(page)
+    gui.run()
+    ```
 
-To create a *Partial*, you must call the method `(Gui.)add_partial()^` on the *Gui*
-instance of your application. You must give this function a page definition (a string or
-an instance of `Markdown^` or `Html^`), and it returns an instance of `Partial^` that can
-be used in visual elements that use them.
+If you run this Python script and connect a browser to the web server address
+(usually *localhost:5000*), you can see your title displayed in a blank page.
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/index.md]:
```suggestion
(usually *localhost:5000*), you can see your title displayed on am empty page.
```

File: docs/manuals/userman/gui/pages/navigate/index.md [hunk 13]
```diff
@@ -0,0 +1,285 @@
...
+    pages = {
+        "/": root_page,
+        "home": home_page,
+        "about": about_page
+    }
+
+    Gui(pages=pages).run()
+    ```
+
+# Using Menu
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/navigate/index.md]:
```suggestion
# Using menus
```

File: docs/manuals/userman/gui/pages/navigate/index.md [hunk 14]
```diff
@@ -0,0 +1,285 @@
...
+
+[Menu](../../viselements/generic/menu.md) creates a menu on the left to
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/navigate/index.md]:
```suggestion
The [`menu`](../../viselements/generic/menu.md) control displays a menu to the left of the page, allowing to
```

File: docs/manuals/userman/gui/pages/navigate/index.md [hunk 15]
```diff
@@ -0,0 +1,285 @@
...
+navigate through the pages.
+
+=== "Markdown"
+    ```
+    <|menu|label=Menu|lov={lov_pages}|on_action=on_menu|>`
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/navigate/index.md]:
```suggestion
    <|menu|label=Menu|lov={lov_pages}|on_action=menu_option_selected|>`
```

File: docs/manuals/userman/gui/pages/navigate/index.md [hunk 16]
```diff
@@ -0,0 +1,285 @@
...
+    ```
+=== "Python"
+    ```python
+    tgb.menu(label="Menu", lov=[...], on_action=on_menu)
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/navigate/index.md]:
```suggestion
    tgb.menu(label="Menu", lov=[...], on_action=menu_option_selected)
```

File: docs/manuals/userman/gui/pages/navigate/index.md [hunk 17]
```diff
@@ -0,0 +1,285 @@
...
+![Menu](images/menu.png){ width=40% : .tp-image-border }
+
+For example, this code creates a menu with two options:
+
+=== "Markdown"
+    ```python
+    from taipy import Gui, navigate
+
+    root_md = """
+    <|menu|label=Menu|lov={[('home', 'Home'), ('about', 'About')]}|on_action=on_menu|>
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/navigate/index.md]:
```suggestion
    <|menu|label=Menu|lov={[('home', 'Home'), ('about', 'About')]}|on_action=menu_option_selected|>
```

File: docs/manuals/userman/gui/pages/navigate/index.md [hunk 18]
```diff
@@ -0,0 +1,285 @@
...
+    # Multi-page application
+    """
+
+    home_md = "# Home"
+    about_md = "# About"
+
+    def on_menu(state, action, info):
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/navigate/index.md]:
```suggestion
    def menu_option_selected(state, action, info):
```

File: docs/manuals/userman/gui/pages/navigate/index.md [hunk 19]
```diff
@@ -0,0 +1,285 @@
...
+    }
+
+    Gui(pages=pages).run()
+    ```
+=== "Python"
+    ```python
+    from taipy import Gui, navigate
+    import taipy.gui.builder as tgb
+
+    def on_menu(state, action, info):
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/navigate/index.md]:
```suggestion
    def menu_option_selected(state, action, info):
```

File: docs/manuals/userman/gui/pages/navigate/index.md [hunk 20]
```diff
@@ -0,0 +1,285 @@
...
+        page = info["args"][0]
+        navigate(state, to=page)
+
+    with tgb.Page() as root_page:
+        tgb.menu(label="Menu",
+                lov=[('home', 'Page 1'), ('about', 'Page 2')],
+                on_action=on_menu)
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/navigate/index.md]:
```suggestion
                on_action=menu_option_selected)
```

File: docs/manuals/userman/gui/pages/navigate/index.md [hunk 21]
```diff
@@ -0,0 +1,285 @@
...
+        "home": home_page,
+        "about": about_page
+    }
+
+    Gui(pages=pages).run()
+    ```
+
+# Using the `navigate` function
+
+The `navigate` function allows for programmatically controlling navigation
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/navigate/index.md]:
```suggestion
The `(Gui.)navigate()^` function allows for programmatically controlling navigation 
```

File: docs/manuals/userman/gui/pages/navigate/index.md [hunk 22]
```diff
@@ -0,0 +1,285 @@
...
+        "/": root_page,
+        "home": home_page,
+        "about": about_page
+    }
+
+    Gui(pages=pages).run()
+    ```
+
+
+# Using the `on_navigate` function
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/navigate/index.md]:
```suggestion
# Using the `on_navigate` callback
```

File: docs/manuals/userman/gui/pages/navigate/index.md [hunk 23]
```diff
@@ -0,0 +1,285 @@
...
+
+The `on_navigate` function allows for custom control over navigation, such
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/navigate/index.md]:
```suggestion
The `on_navigate` callback allows for customizing the control over navigation, such 
```

File: docs/manuals/userman/gui/pages/navigate/index.md [hunk 24]
```diff
@@ -0,0 +1,285 @@
...
+    Gui(pages=pages).run()
+    ```
+
+
+# Using the `on_navigate` function
+
+The `on_navigate` function allows for custom control over navigation, such 
+as redirecting users based on conditions.
+
+See more information for [*on_navigate*](../../callbacks.md#navigation-callback).
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/navigate/index.md]:
```suggestion
See more information for [`on_navigate`](../../callbacks.md#navigation-callback).
```

File: docs/manuals/userman/gui/pages/navigate/index.md [hunk 25]
```diff
@@ -0,0 +1,285 @@
...
+    results_md = "# Results Page"
+    no_results_md = "# No Results Available"
+
+    pages = {
+        "/": root_md,
+        "results": results_md,
+        "no_results": no_results_md
+    }
+
+    gui = Gui(pages=pages)
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/navigate/index.md]:
```suggestion
    gui = Gui(pages=pages).run()
```

File: docs/manuals/userman/gui/pages/navigate/index.md [hunk 26]
```diff
@@ -0,0 +1,285 @@
...
+    gui.run()
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/navigate/index.md]:
```suggestion
```

File: docs/manuals/userman/gui/pages/navigate/index.md [hunk 27]
```diff
@@ -0,0 +1,285 @@
...
+    with tgb.Page() as no_results_page:
+        tgb.text("# No Results Available", mode="md")
+
+    pages = {
+        "/": root_page,
+        "results": results_page,
+        "no_results": no_results_page
+    }
+
+    gui = Gui(pages=pages)
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/navigate/index.md]:
```suggestion
    gui = Gui(pages=pages).run()
```

File: docs/manuals/userman/gui/pages/navigate/index.md [hunk 28]
```diff
@@ -0,0 +1,285 @@
...
+    gui.run()
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/navigate/index.md]:
```suggestion
```

File: docs/manuals/userman/gui/pages/partial/index.md [hunk 29]
```diff
@@ -1,22 +1,88 @@
...
+
+# Realtime update of partials
 
 Taipy's partials feature allows you to create reusable and dynamically updatable 
 components in your GUI applications. This is particularly useful for creating consistent 
-layouts and updating content on the fly. For example, you can create a list of links that 
-can be dynamically updated without reloading the entire page.
+layouts and updating content on the fly. 
+
+You are able to update your partial by using the `update_content` method.
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/partial/index.md]:
```suggestion
You are able to update your partial by using the `(Partial.)update_content()^` method.
```

File: docs/manuals/userman/gui/pages/partial/index.md [hunk 30]
```diff
@@ -1,22 +1,88 @@
...
 
-![Partials Example](images/illustration.png){width=50% : .tp-image-border}
+```python
+state.partial.update_content(state, new_content)
+```
 
-In this article, we will walk through a simple example where:
+The `update_content` method takes two arguments: the state object and the new content of the partial.
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/partial/index.md]:
```suggestion
The `(Partial.)update_content()^` method expects two arguments: the state object and the new content of the partial.
```

File: docs/manuals/userman/gui/pages/partial/index.md [hunk 31]
```diff
@@ -1,22 +1,88 @@
...
 
-- A main page displays a list of links.
-- The list can be dynamically updated with new links using a button click.
+=== "Markdown"
+    ```python linenums="1"
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/partial/index.md]: Why the line numbers??

File: docs/manuals/userman/gui/pages/partial/index.md [hunk 32]
```diff
@@ -1,22 +1,88 @@
...
+    <|Change partial|button|on_action=change_partial|>
+    """
+
+    # Create and run the Taipy GUI
+    gui = Gui(page)
+    partial = gui.add_partial("Your first partial")
+    gui.run()
+    ```
+=== "Python"
+    ```python linenums="1"
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/partial/index.md]: Why the line numbers??

File: docs/manuals/userman/gui/pages/partial/index.md [hunk 33]
```diff
@@ -59,7 +124,7 @@ gui.run()
 Here, `link_partial` is declared with an empty string. This partial will be dynamically
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/partial/index.md]: What is this about?

File: docs/manuals/userman/gui/pages/partial/index.md [hunk 34]
```diff
@@ -59,7 +124,7 @@ gui.run()
 Here, `link_partial` is declared with an empty string. This partial will be dynamically 
 updated later.
 
-## Step 2: Using the Partial in the Layout
+### Using the Partial in the Layout
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/partial/index.md]: Why the capitals??

File: docs/manuals/userman/gui/pages/partial/index.md [hunk 35]
```diff
@@ -82,16 +147,15 @@ include it in one or multiple places within your layout.
 In this code, `tgb.part(partial="{link_partial}")` is where the partial will be shown. 
 You can place this partial in multiple locations within your layout if needed.
 
-## Step 3: Updating the Partial Content
+### Updating the Partial Content
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/partial/index.md]: Why the capitals??

File: docs/tutorials/fundamentals/5_rag_chatbot/index.md [hunk 36]
```diff
@@ -131,7 +131,7 @@ We begin by defining the following [state variables](../../../manuals/userman/gu
 1. `query_message`: A string bound to an input box, for the user to enter their query for the model;
```
Comment (FabienLelaquais) [inline docs/tutorials/fundamentals/5_rag_chatbot/index.md]:
```suggestion
1. *query_message*: A string bound to an input box, for the user to enter their query for the model;
```

File: docs/tutorials/fundamentals/5_rag_chatbot/index.md [hunk 37]
```diff
@@ -131,7 +131,7 @@ We begin by defining the following [state variables](../../../manuals/userman/gu
...
 2. `messages`: A list of all messages between the user and LLM;
```
Comment (FabienLelaquais) [inline docs/tutorials/fundamentals/5_rag_chatbot/index.md]:
```suggestion
2. *messages*: A list of all messages between the user and LLM;
```

File: docs/tutorials/fundamentals/5_rag_chatbot/index.md [hunk 38]
```diff
@@ -131,7 +131,7 @@ We begin by defining the following [state variables](../../../manuals/userman/gu
...
 3. `messages_dict`: A dictionary used to hold the chat partial content (see next step); and
```
Comment (FabienLelaquais) [inline docs/tutorials/fundamentals/5_rag_chatbot/index.md]:
```suggestion
3. *messages_dict*: A dictionary used to hold the chat partial content (see next step); and
```

File: docs/tutorials/fundamentals/5_rag_chatbot/index.md [hunk 39]
```diff
@@ -131,7 +131,7 @@ We begin by defining the following [state variables](../../../manuals/userman/gu
...
-4. `conv`: A Taipy [*partial*](../../../manuals/userman/gui/pages/index.md#partials) whose content is updated as new chat
+4. `conv`: A Taipy [*partial*](../../../manuals/userman/gui/pages/partial/index.md) whose content is updated as new chat
```
Comment (FabienLelaquais) [inline docs/tutorials/fundamentals/5_rag_chatbot/index.md]:
```suggestion
4. *conv*: A [`partial`](../../../manuals/userman/gui/pages/partial/index.md) whose content is updated as new chat
```

File: mkdocs.yml_template [hunk 40]
```diff
@@ -81,7 +79,13 @@ nav:
           - "Styling":
               - "Styling": manuals/userman/gui/styling/index.md
               - "The Stylekit": manuals/userman/gui/styling/stylekit.md
-        - "Pages": manuals/userman/gui/pages/index.md
+        - "Pages":
+          - "Pages": manuals/userman/gui/pages/index.md
+          - "Navigate Between Pages": manuals/userman/gui/pages/navigate/index.md
```
Comment (FabienLelaquais) [inline mkdocs.yml_template]: Why the capitals?

File: mkdocs.yml_template [hunk 41]
```diff
@@ -81,7 +79,13 @@ nav:
...
+          - "Content update with Partials": manuals/userman/gui/pages/partial/index.md
```
Comment (FabienLelaquais) [inline mkdocs.yml_template]: Why the capitals?

File: mkdocs.yml_template [hunk 42]
```diff
@@ -81,7 +79,13 @@ nav:
...
+          - "Dialog": manuals/userman/gui/pages/dialog/index.md
```
Comment (FabienLelaquais) [inline mkdocs.yml_template]:
```suggestion
          - "Dialogs": manuals/userman/gui/pages/dialog/index.md
```

File: mkdocs.yml_template [hunk 43]
```diff
@@ -81,7 +79,13 @@ nav:
...
+          - "Pane": manuals/userman/gui/pages/pane/index.md
```
Comment (FabienLelaquais) [inline mkdocs.yml_template]:
```suggestion
          - "Panes": manuals/userman/gui/pages/pane/index.md
```

**Review by FabienLelaquais — CHANGES_REQUESTED** [review]
Quite a few typographic inconsistencies.

File: docs/manuals/userman/gui/pages/index.md (hunk 10 above)
Comment (FlorianJacta) [reply docs/manuals/userman/gui/pages/index.md]: I am not sure to understand this. I am showing the two ways to add pages. I fixed the code btw

File: docs/manuals/userman/gui/pages/navigate/index.md (hunk 25 above)
Comment (FlorianJacta) [reply docs/manuals/userman/gui/pages/navigate/index.md]: Not sure about this one too. Do you want to erase the `gui = `then too?

File: docs/manuals/userman/gui/pages/partial/index.md (hunk 33 above)
Comment (FlorianJacta) [reply docs/manuals/userman/gui/pages/partial/index.md]: I don't understand your comment

**Comment by FlorianJacta** [comment]
I have accepted your suggestions except for two remarks @FabienLelaquais

File: docs/manuals/userman/gui/pages/partial/index.md (hunk 33 above)
Comment (FabienLelaquais) [reply docs/manuals/userman/gui/pages/partial/index.md]: What is `link_partial`? A class?
If it's a variable, it should be \*link_partial\*. And where is it declared or defined?

File: docs/manuals/userman/gui/pages/navigate/index.md [hunk 44]
```diff
@@ -166,7 +166,7 @@ For example, this code creates a menu with two options:
 
 # Using the `navigate` function
 
-The `navigate` function allows for programmatically controlling navigation 
+The `(Gui.)navigate()^` allows for programmatically controlling navigation
```
Comment (FabienLelaquais) [inline docs/manuals/userman/gui/pages/navigate/index.md]:
```suggestion
The `(Gui.)navigate()^` function allows for programmatically controlling navigation 
```

**Review by FabienLelaquais — APPROVED** [review]
Tiny typo...

File: docs/manuals/userman/gui/pages/index.md (hunk 10 above)
Comment (FabienLelaquais) [reply docs/manuals/userman/gui/pages/index.md]: I see no point (at all) to split `Gui(page).run()` into three lines.
And homogeneity is important.

File: docs/manuals/userman/gui/pages/partial/index.md (hunk 33 above)
Comment (FlorianJacta) [reply docs/manuals/userman/gui/pages/partial/index.md]:
```python
gui = Gui(main_page)
link_partial = gui.add_partial("")
gui.run()
```

Here, *link_partial* is declared with an empty string. This partial will be dynamically 
updated later.

Comment (FlorianJacta) [reply docs/manuals/userman/gui/pages/partial/index.md]: It is declared just above. I can put the line numbers. I have put it between `*`

**Pull request merged**